apps/src/code-studio/components/stageExtras/BonusLevels.jsx

```diff
@@ -15,6 +17,10 @@ const styles = {
     clear: 'both',
     overflow: 'hidden',
   },
+  solutionImage: {
+    border: '1px solid black',
+    marginBottom: 5,
+  }
```

Comment:
Good catch Josh, artist and playlab thumbnails are in fact bigger due to the border. I included a border because the artist levels look bad without any deliniation between the thumbnail background and the page background. I'm adding the same border to the maze thumbnails for now. Here's how they look side by side now:
<img width="846" alt="screen shot 2017-08-07 at 2 05 08 pm" src="https://user-images.githubusercontent.com/8001765/29045769-98a494dc-7b79-11e7-9d3a-824339e3caf3.png">